properties/basic.py

```diff
@@ -4,11 +4,15 @@
 from __future__ import unicode_literals
 
 import datetime
+from io import BytesIO
 from uuid import uuid4
+
+import numpy as np
+from png import Reader
```

Comment:
I am not sure I am happy about this being a dependency of this repository. Can we move the Image to a separate repo?